src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/block/mirroring/mirroring.component.html

```diff
@@ -27,8 +28,7 @@
 
       <cd-table [data]="pools.data"
                 columnMode="flex"
-                [columns]="pools.columns"
-                (fetchData)="refresh()">
```

Comment:
This is needed for us to use the refresh button. Same of the other tables.

Reply:
Yes this is an interesting case, because you need a way to trigger the fetch function but it should never be triggered, I guess I will change the usage of no auto reload, so that it never automatically loads it.